### ISSUE TITLE
Add zooming

### ISSUE THREAD

**Issue by lemmy** [abstratt/eclipsegraphviz#4]
Assuming the rendered graph is large, zooming in/out would be a nice addition.

**Comment by abstratt** [comment]
Indeed. To be completely honest though, it is probably not going to happen unless someone sends a pull request, I won't be able to work on this feature myself.

**Comment by thoorp** [comment]
svg-Pan-zoom worked for me.

**Comment by lemmy** [comment]
@thoorp I have no idea what "svg-Pan-zoom" is supposed to mean.

**Comment by thoorp** [comment]
Used the below library to implement zoom in/out on top of svg.

https://github.com/ariutta/svg-pan-zoom

Hope that helps!

**Comment by ronatartifact** [comment]
The code seems to indicate that the output format can be selected but I can not find where to say that I want SVG rather than JPG. This will solve the zoom problem since the output file will support zoom.

**Comment by abstratt** [comment]
@ronatartifact You choose the format by specifying a file with the desired file extension. These are the supported formats and their extensions:

JPEG("jpg"), PNG("png"), GIF("gif"), TIFF("tif"), BITMAP("bmp"), SVG("svg"), DOT("dot")
